### PR TITLE
Build Image and Input page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview Link : https://deploy-preview-4--felizui.netlify.app/

### CODE REVIEW

File: pages/input.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles/page.css">
```
Comment (ranitasahaa1230) [inline pages/input.html:10]: You can use author  name and description details in your meta tag

**Review by ranitasahaa1230 — CHANGES_REQUESTED** [review]
You can use author and description name in your meta tag

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me.

File: pages/input.html (hunk 1 above)
Comment (ishitakeshawani) [reply pages/input.html:10]: Yess Added.

**Pull request merged**